rpcgen/rpc_main.c

```diff
@@ -348,6 +348,12 @@ find_cpp (void)
       crash ();
     }
 
+  if ((CPP = getenv("RPCGEN_CPP")) != NULL)
+    return;
+
+  if ((CPP = getenv("CPP")) != NULL)
+    return;
+
```

Comment:
Like with the user provided cpp path (via -Y argument) some lines above, it should be checked that the environment variable is correct and cpp exists.